CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+Creating a New Example Implementation
+============
+
+Getting Started
+---
+1. [Fork this repo](https://github.com/ericelliott/todotasks/fork)
+
+2. clone your fork and copy the `base` files
```

Comment:
We really need to make a script to make this easier. See https://github.com/ericelliott/todotasks/issues/12.